nav2_dijkstra_planner/README.md

```diff
@@ -0,0 +1,25 @@
+# Dijkstra Planner
+The DijkstraPlanner is an implementation of a [Planning module](../doc/requirements/requirements.md) (equivalent to GlobalPlanner in ROS1 [MoveBase](http://wiki.ros.org/move_base)).
+
+The Planning module is responsible for generating a feasible path given start and end robot poses.
+
+Dijkstra’s algorithm is guaranteed to find the shortest path under any condition.
+
+## Status
+Currently, DijkstraPlanner is a direct port from ROS1 MoveBase [Navfn](http://wiki.ros.org/navfn) planner.
+
+The Navfn planner assumes a circular robot and operates on a costmap to find a minimum cost plan from a start point to an end point in a grid. The navigation function is computed with Dijkstra's algorithm. The strategy is based on [High-Speed Navigation Using
```

Comment:
Doesn't need a paragraph break here since the topic is still Navfn.

Reply:
Will change this.